plugins/scaffolder-backend/src/scaffolder/actions/builtin/github/githubActionsDispatch.ts

```diff
@@ -68,18 +69,31 @@ export function createGithubActionsDispatchAction(options: {
               'Inputs keys and values to send to GitHub Action configured on the workflow file. The maximum number of properties is 10. ',
             type: 'object',
           },
+          token: {
+            title: 'Authentication Token',
+            type: 'string',
+            description: 'The GITHUB_TOKEN to use for authorization to GitHub',
```

Comment:
```suggestion
            description: 'The token to use for authorization to GitHub',
```

Bit worried about causing confusion around this being some kind of constant. Same for all of the other occurrences of this too

Reply:
Hmm yeah good point - I was just thinking that it seemed most familiar, but I get the confusion will update for all providers.